### PR TITLE
EntityName : add prefix and agent host replacement 

### PR DESCRIPTION
**Pull request by alvarocabanas**
- Modify entityName to use agent "localhost" autoreplacement and add prefix "win_service"

### CODE REVIEW

File: src/nri/metricsProcesor.go [hunk 1]
```diff
@@ -18,34 +18,36 @@ import (
 	dto "github.com/prometheus/client_model/go"
 )
 
+const (
+	entityNamePrefix = "win_service"
```
Comment (gsanchezgavier) [inline src/nri/metricsProcesor.go:22]: why not `WIN_SERVICE` like the entity type itself ?

File: src/nri/metricsProcesor.go [hunk 2]
```diff
@@ -18,34 +18,36 @@ import (
...
+	// localhost will be automatically replaced by the host_name or display_name by the agent
```
Comment (gsanchezgavier) [inline src/nri/metricsProcesor.go:23]:
```suggestion
	// localhost will be automatically replaced by the host_name or display_name by the agent when is found inside EntityName
```

File: src/nri/metricsProcesor.go (hunk 1 above)
Comment (alvarocabanas) [reply src/nri/metricsProcesor.go:22]: What do you mean, in capital letters?

Comment (alvarocabanas) [reply src/nri/metricsProcesor.go]: Ok, I set it up in capital letters

**Review by gsanchezgavier — APPROVED** [review]
lgtm

**Pull request merged**